scripts/asm-installer/install_asm

```diff
@@ -728,6 +733,47 @@ validate_istio_version() {
   fi
 }
 
+validate_asm_version() {
+  info "Checking existing ASM version(s)..."
+  local OUTPUT
+  OUTPUT="$(retry 3 ./istio-${RELEASE}/bin/istioctl version -o json)"
```

Comment:
Can you use ./"${ISTIOCTL_REL_PATH}" instead of istio-whatever? I think it should be set by this point in the script.

Reply:
SG